docs/quickstart.rst

```diff
@@ -3,361 +3,563 @@
+
+
+Depending on your system's horsepower, give them a few seconds/minutes to fully boot-up.
+
+
+Connecting to the containers
+______________________________
+
+At this stage, you should be able to view the **Web Console** by navigating to:
 
-Now that we have the needed docker files, we will need to get the `Inmanta quickstart project <https://github.com/inmanta/quickstart>`_ itself:
+http://172.30.0.3:8888/console
+
+To get an interactive shell to the Inmanta server:
 
 .. code-block:: sh
 
-    git clone https://github.com/inmanta/quickstart.git quickstart-project
+    docker exec -it clab-srlinux-inmanta-server /bin/bash
 
-The quickstart project can now be found under the newly created `quickstart-project` directory.
-It will be the basis for this quickstart.
-The ``quickstart-project`` directory will also be shared with the Inmanta server container
-(mounted to ``/home/inmanta/quickstart-project``).
-We will come back to the files in this repository later.
 
-.. note::
+In order to connect to `SR Linux` containers, there are two options:
+
+1. Using Docker:
+
+.. code-block:: sh
 
-    If you are on `Windows`, be sure you make the drive with the quickstart project shareable with docker containers:
+    docker exec -it clab-srlinux-spine sr_cli
+    # or
+    docker exec -it clab-srlinux-leaf1 sr_cli
+    # or
+    docker exec -it clab-srlinux-leaf2 sr_cli
 
-    1. In Powershell: ``$env:COMPOSE_CONVERT_WINDOWS_PATHS = 1``
-    2. Restart Docker for Windows
-    3. Go to Docker for Windows settings > Shared Drives > Reset credentials > select drive with quickstart project > set your credentials > Apply
 
-Finally, have Docker Compose deploy the quickstart environment:
+2. Using SSH (username and password is `admin`):
 
 .. code-block:: sh
 
-    docker-compose up
+   ssh admin@clab-srlinux-spine
+   ssh admin@clab-srlinux-leaf1
+   ssh admin@clab-srlinux-leaf2
 
-Docker Compose will set up the Inmanta server, a postgres server and two VM-like containers to experiment on.
-When Docker Compose is done deploying and the Inmanta server is running, you will be able to open the dashboard at http://127.0.0.1:8888/dashboard.
-When you see the following output, the Inmanta server is ready to be used:
+Then enter the `configuration mode` by typing:
 
 .. code-block:: sh
 
-    inmanta_quickstart_server | inmanta.protocol.rest    DEBUG   Start REST transport
-    inmanta_quickstart_server | inmanta                  INFO    Server startup complete
+    enter candidate
 
-.. note::
+The output should look something like this:
+
+.. code-block::
+
+    Welcome to the srlinux CLI.
+    Type 'help' (and press <ENTER>) if you need any help using this.
 
-    docker-compose will lock the current terminal and use it for output from all 4 containers.
-    You will need to open a new terminal to continue with this quickstart
 
-To get an interactive shell on the Inmanta server (this will be needed later):
+    --{ running }--[  ]--
```

Comment:
When trying the command in spine, I do not observe this output. Instead I get "--{ candidate shared default }--[  ]--" (without quotes)

Reply:
Could you verify from the console that the agent actually pushed the config to the spine?

Reply:
But that this before the orchestrator actually does anything, isn't it?
 The issue reported here is I think that when you type `enter candidate` the cli doesn't show `--{ running }--[  ]--` but `--{ candidate shared default }--[ ]--` (which seems correct to me)

Reply:
I see, good catch!